tests/bluetooth/mesh_shell/src/main.c

```diff
@@ -18,11 +18,44 @@ static struct bt_mesh_cfg_cli cfg_cli;
 
 BT_MESH_SHELL_HEALTH_PUB_DEFINE(health_pub);
 
+static void health_current_status(struct bt_mesh_health_cli *cli, uint16_t addr,
```

Comment:
Since this is test I'd expect to see more detail information in shell that includes function incoming parameters for status messages.